### PR TITLE
Added new sample

### PR DESCRIPTION
**Pull request by singhgag**
- List users in group

Use build 1486

### CODE REVIEW

**Comment by dg0yal** [comment]
Just the one comment. Otherwise looks good. Nice !

**Comment by singhgag** [comment]
@dg0yal I don't see your comment. Can you point me to it?

**Comment by dg0yal** [comment]
See it now?

File: arcgis-ios-sdk-samples/Cloud & Portal/List users in group/ListUsersInGroupVC.swift [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        
+        //fetch users in the group
+        self.portalGroup.fetchUsersWithCompletion { [weak self] (usernames: [String]?, admins: [String]?, error: NSError?) in
+            if let error = error {
+                SVProgressHUD.showErrorWithStatus(error.localizedDescription, maskType: .Gradient)
+            }
+            else {
+                if let usernames = usernames where usernames.count > 0, let weakSelf = self {
+                    //if usernames are found create AGSPortalUser objects for each
+                    let portalUsers = weakSelf.portalUsersFor(usernames)
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Cloud & Portal/List users in group/ListUsersInGroupVC.swift]: Ok, then you can merge the branch as is after you resolve the conflicts

**Pull request closed without merging**